librz/analysis/analysis.c

```diff
@@ -563,7 +563,7 @@ static bool noreturn_recurse(RzAnalysis *analysis, ut64 addr) {
 	ut8 bbuf[0x10] = { 0 };
 	ut64 recurse_addr = UT64_MAX;
 	if (!analysis->iob.read_at(analysis->iob.io, addr, bbuf, sizeof(bbuf))) {
-		eprintf("Couldn't read buffer\n");
+		RZ_LOG_ERROR("Couldn't read buffer\n");
```

Comment:
You used `Cannot` instead of `Can't`, probably makes sense to do it here as well.